python/ray/_private/gpu_object_manager.py

```diff
@@ -79,6 +76,19 @@ def __ray_get_tensor_meta__(self, obj_id: str):
 
         return src_actor.__ray_call__.remote(__ray_get_tensor_meta__, obj_id)
 
+    def is_managed_gpu_object(self, obj_id: str) -> bool:
```

Comment:
Seems this is nearly a duplicate of _is_gpu_object_ref. Can we have just one?

Reply:
oh sorry, I didn't notice that.